news/12782.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Improve performance of pip install. When extracting a wheel,
+do not recheck/recreate the parent directory before extracting each file.
```

Comment:
```suggestion
Improve pip install performance by only creating required parent
directories once, instead of before extracting every file in the wheel. 
```